src/iam/server/iamserver.cpp

```diff
@@ -76,6 +76,19 @@ void ConvertToProto(const Array<StaticString<cSubjectIDLen>>& src, iamanager::v4
     }
 }
 
+static const std::string CorrectAddress(const std::string& addr)
+{
+    if (addr.empty()) {
+        throw std::runtime_error("bad address");
```

Comment:
Will this exception be handled in Init?

Reply:
Init method handles std::exception, which runtime_error is a subclass of.